web/src/components/drawer/netflow-traffic-drawer.tsx

```diff
@@ -148,28 +147,21 @@ export const NetflowTrafficDrawer: React.FC<NetflowTrafficDrawerProps> = React.f
       [props.setFilters]
     );
 
-    const filterLinks = React.useCallback(() => {
-      const defFilters = props.defaultFilters;
-      return (
-        <LinksOverflow
-          id={'filter-links-overflow'}
-          items={[
-            {
-              id: 'reset-filters',
-              label: t('Reset defaults'),
-              onClick: props.resetDefaultFilters,
-              enabled: defFilters.length > 0 && !filtersEqual(props.filters.list, defFilters)
-            },
-            {
-              id: 'clear-all-filters',
-              label: t('Clear all'),
-              onClick: props.clearFilters,
-              enabled: props.filters.list.length > 0
-            }
-          ]}
-        />
-      );
-    }, [props.defaultFilters, props.resetDefaultFilters, props.filters.list, props.clearFilters]);
+    const getResetDefaultFiltersProp = React.useCallback(() => {
+      if (props.defaultFilters.length > 0 && !filtersEqual(props.filters.list, props.defaultFilters)) {
+        return props.resetDefaultFilters;
+      }
+      return undefined;
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why disable linter, it seems to have already all the needed deps?

Reply:
Without that, the linter expect `props` to be the dependency here and it will refresh on every change.

Reply:
we can destructure props (just the bits we need) to get them back: https://github.com/netobserv/network-observability-console-plugin/pull/589/commits/b42aacae210a95fb9d82c354dc58ccbe3a1f819a